openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -181,6 +254,13 @@ def test_patch_account_different_user(self):
         self.different_client.login(username=self.different_user.username, password=TEST_PASSWORD)
         self.send_patch(self.different_client, {}, expected_status=404)
 
+    def test_patch_account_is_staff(self):
```

Comment:
That sounds good, thanks. Another options is to do DDT for all three if you also supplied the expected return code, I suppose. I find that it is easier to read that the tests are comprehensive if there is one test that verifies the states. It is hard to read around the code and make sure that all the variations are done.